### DISCUSSION TITLE
14 security vendors and 1 sandbox flagged this file as malicious

### DISCUSSION THREAD

**Discussion by satishuidev** [CzBiX/WSLHostPatcher#16] [General]
I started using this since a month ago. Suddenly our Cybersecurity pinged me mentioned that they received an alert from my machine as they have seen a potential malicious file in my system. I have uninstalled it. Here is the report
https://www.virustotal.com/gui/file/5b0e71fea033b70f0655031e212e5e05febcb5719cbe8ef0e518c8ca5e8a93d3/detection

I deleted it now and using [WSL2_VPN_Workaround_Instructions.md] now

**Comment by CzBiX** [comment]
This was completely expected due to the technology used in the project.
All code and libraries in the project are open source, so if you have concerns, you can review them yourself and compile them for use.

**Comment by sglbl** [comment]
Now Windows also gives security warning:

![image](https://github.com/CzBiX/WSLHostPatcher/assets/64928475/6bc6c822-6ccc-47d8-8d93-2d7209ef0347)

**Comment by rfabro** [comment]
Encountered the same thing. Windows Defender flagged WSLHostPatch.dll as trojan

![image](https://github.com/user-attachments/assets/f5e0a169-3aab-4682-b14c-e947d4f0280a)
